.github/workflows/desktop.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         os: [windows-latest, ubuntu-latest, macos-latest]
         build_type: ["Release", "Debug"]
         architecture: ["x64",]
-        python_version: [2.7]
+        python_version: [3.6]
         generator: ["Visual Studio 16 2019", "Unix Makefiles", "Ninja"]
```

Comment:
"Ninja" was there just to make sure that things work with Ninja incase users are using it. 
If we decide to not run our workflows against "Ninja", we can get rid of this `generator`. And we dont have to do aynthing in the build script because cmake will pick the default generator based on the platform.

Reply:
Oh and I was curious to know if Ninja builds faster than other generators. So yeah, we could technically take Ninja off the list and that simplifies the build script too. Will still need an additional step of installing ccache though, else we could do everything locally in the repo with vcpkg and not touch any system level packages whatsoever.

Reply:
Yeah I think since Ninja doesn't make a difference, i'm fine with just testing against the default. If a developer wants to use Ninja they can be responsible for whether it works themselves, I'd rather not have to support more than 1 generator per platform.